launcher/lwjre/lib/calendars.properties

```diff
@@ -21,40 +21,36 @@
 #
```

Comment:
Are you *sure* you want to touch this (or anything else in the lwjre directory)? This is a) not created by us and b) some sort of weird config format that might be broken by wrong spacing